mkosi/__init__.py

```diff
@@ -1432,6 +1432,7 @@ def configure_dracut(args: CommandLineArguments, root: str) -> None:
     ):
         with open(os.path.join(dracut_dir, "30-mkosi-uefi-stub.conf"), "w") as f:
             f.write("uefi_stub=/usr/lib/systemd/boot/efi/linuxx64.efi.stub\n")
+            f.write("add_drivers+=efivarfs\n")
```

Comment:
The other config files enclose dracut lists in quotes with spaces. If possible, this should do that as well.

Reply:
fixed